lib/importer/src/dudk/backend.js

```diff
@@ -1,14 +1,24 @@
+const fs = require("fs")
 const xlsx = require("xlsx");
 const crypto = require("crypto");
 const assert = require('node:assert').strict;
 const attributeTypes = require('./types/attribute-types');
+const store = require("./session_store")
 
 // An implementation of the interface described in https://struct.register-dynamics.co.uk/trac/wiki/DataImporter/API
 
-// FIXME: State storage all in-memory for now...
+let sessionStore = function () {
+  const diskFolder = process.env.DUDK_SESSION_STORE_DISK
```

Comment:
Given that we always clean up any on-disk files when the process closes, is there any problem with just setting this to a temporary directory by default, and/or controlling the store used by a switch rather than having to manually set a path?

And/or, is it worth defaulting to on-disk when NODE_ENV is development mode (e.g. npm run dev), but using memory otherwise?